eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/configuration/ImportDbsnpVariantsJobConfigurationFailValidationStepTest.java

```diff
@@ -56,6 +67,15 @@ public void executeJobTrueForceImport() throws Exception {
     public void executeJobFalseForceImport() throws Exception {
         inputParameters.setForceImport("false");
         JobExecution jobExecution = jobLauncherTestUtils.launchJob();
-        assertEquals(BatchStatus.COMPLETED, jobExecution.getStatus());
+
+        List<String> expectedSteps = Collections.singletonList(VALIDATE_CONTIGS_STEP);
+        assertStepsExecuted(jobExecution, expectedSteps);
+        assertEquals(BatchStatus.FAILED, jobExecution.getStatus());
+    }
+
+    private void assertStepsExecuted(JobExecution jobExecution, List expectedSteps) {
```

Comment:
I know it's a bit pedantic, but could you please reorder the arguments so the expected steps go first, like in JUnit methods? Applies to the other test class too.

Reply:
I like that approach you suggested better!